resources/roadmap/index.md

```diff
@@ -214,23 +214,34 @@ meta_descr: meta_descr.roadmap
                                     <li class="completed">{% t roadmap.genimprov %}</li>
                             </ul>
                     </div>
+                    <label for="tab-6" tabindex="0"></label>
+                    <input id="tab-6" type="radio" name="tabs" aria-hidden="true" checked>
+                    <h2>2024</h2>
+                    <div class="tabPanel-content info-block">
+                        <h2>2024</h2>
+                            <ul>
+                                <h3 class="months">{% t roadmap.jan %}</h3>
+                                    <li class="completed">{% t roadmap.xmrbch-atomicswap %}</li>
+                                <h3 class="months">{% t roadmap.may %}</h3>
+                                    <li class="completed">{% t roadmap.haveno-mainnet %}</li>
+                            </ul>
+                    </div>
                     <label for="tab-7" tabindex="0"></label>
                     <input id="tab-7" type="radio" name="tabs" aria-hidden="true">
                     <h2>{% t roadmap.future %}</h2>
                     <div class="tabPanel-content info-block">
                         <h2>{% t roadmap.future %}</h2>
                             <ul>
                                 <h3 class="months">{% t roadmap.comingsoon %}</h3>
-                                    <li class="ongoing"><a href="https://haveno.exchange">{% t roadmap.haveno %}</a></li>
-                                    <li class="ongoing"><a href="https://github.com/seraphis-migration/strategy/wiki">{% t roadmap.seraphis-jamtis %}</a></li>
                                     <li class="ongoing"><a href="https://github.com/Rucknium/OSPEAD">{% t roadmap.ospead %}</a></li>
-                                    <li class="ongoing">{% t roadmap.ringsize-sera-tryp %}</li>
-                                    <li class="ongoing">{% t roadmap.tryptych %}</li>
                                     <li class="ongoing">{% t roadmap.xmr-eth-atomicswaps %}</li>
                                     <li class="ongoing">{% t roadmap.bp++ %}</li>
                                     <li class="ongoing">{% t roadmap.lws-subaddress %}</li>
-                                    <li class="upcoming">{% t roadmap.returnaddr %}</li>
-                                    <li class="upcoming">{% t roadmap.layer2 %}</li>
+                                    <li class="ongoing">{% t roadmap.cuprate %}</li>
+                                    <li class="upcoming">{% t roadmap.tryptych %}</li>
```

Comment:
Remove

Reply:
seems sane. using this as a reference: see 'proposal closed' https://ccs.getmonero.org/proposals/cypherstack-sarang-triptych-research.html